tiatoolbox/tools/stainnorm/reinhard_colour_normaliser.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import cv2
+
+
+class ReinhardColourNormaliser(object):
+    """Normalize a patch colour to the target image using the method of:
+
+    Reinhard et al. "Color transfer between images." Computer graphics & applications.
+
+    Examples:
+        >>> from tiatoolbox.tools.stainnorm import reinhard_colour_normaliser
+        >>> norm = reinhard_colour_normaliser.ReinhardColourNormaliser()
+        >>> norm.fit(target_img)
+        >>> trans = norm.transform(src_img)
+
+    """
+
+    def __init__(self):
+        self.target_means = None
+        self.target_stds = None
+
+    def fit(self, target):
+        """Fit to a target image
+
+        Args:
+            target (RGB uint8): target image
+
+        """
+        means, stds = self.get_mean_std(target)
+        self.target_means = means
+        self.target_stds = stds
+
+    def transform(self, img):
+        """Transform an image.
+
+        Args:
+            img (RGB uint8): Input image
+
+        Returns:
+            RGB float: colour normalised RGB image
+
+        """
+        chan1, chan2, chan3 = self.lab_split(img)
+        means, stds = self.get_mean_std(img)
+        norm1 = (
+            (chan1 - means[0]) * (self.target_stds[0] / stds[0])
+        ) + self.target_means[0]
+        norm2 = (
+            (chan2 - means[1]) * (self.target_stds[1] / stds[1])
+        ) + self.target_means[1]
+        norm3 = (
+            (chan3 - means[2]) * (self.target_stds[2] / stds[2])
+        ) + self.target_means[2]
+        return self.merge_back(norm1, norm2, norm3)
+
+    @staticmethod
+    def lab_split(img):
+        """Convert from RGB uint8 to LAB and split into channels.
+
+        Args:
+            img (RGB uint8): Input image
```

Comment:
ndarray instead of RGB.

Reply:
Fixed in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/38af1ea1723fb76a8c553e6468a60ddc6c75c999)